### ISSUE TITLE
[Feature request] Allow an easy way to build a kernel that isn't from a version tagged git clone while keeping the tkg patch version schemes working

### ISSUE THREAD

**Issue by BillFleming** [Frogging-Family/linux-tkg#1057]
When attempting to use some alternate kernel sources where the branch/tag isn't the version number this currently completely breaks the script. One example is here: 
https://gitlab.collabora.com/hardware-enablement/rockchip-3588/linux

On a particular month it might be version 6.12 or 6.13 (but the branch will be just untagged **rk3588**) but the script can't figure it out correctly which version it is so the patch steps would fail.
You can't pull down the branch normally from the tkg script which is ok, but if you pull it down outside of the script then try to run the script on it the version problem still remains.

Basically we need something like a new _version="6.12" variable that is some version override. Then optionally a separate tag/branch variable to pull down the **rk3588** branch if that is needed.

FYI: You can run linux-tkg on arm and get a working bootable arch package if you supply the correct config file to it. :grin: :frog:

**Comment by AdelKS** [comment]
I may have answered your request with #1081
